src/index.js

```diff
@@ -28,36 +28,40 @@ async function getData(code) {
 	const response = await fetch(`${url}?objeto=${code}&mqs=S`);
 	const data = await response.json();
 
-	if (data.erro) {
-		log(`❌ ${data.mensagem}`);
-		return;
+	if (data?.erro) {
+		log(`❌ ${data?.mensagem}`);
+		return null;
 	}
 
 	return data;
 }
 
-async function run() {
-	const code = process.argv[2].toUpperCase();
+export default async function run() {
+	const code = process?.argv[2]?.toUpperCase();
+
+	if (!code) {
+		log(`🖊️ Escreva um código para que a busca seja realizada!`);
+		return null;
+	}
 
 	logEnter(chalk.bold(`📮 ${code}`));
 
 	const data = await getData(code);
 
 	const events = data?.eventos || [];
 
-	events.map((event) => {
+	events?.forEach((event) => {
 		const { descricao, descricaoWeb, dtHrCriado, unidade, unidadeDestino } = event;
 
 		log(`==> ${getIcon(descricaoWeb)} ${descricao}`);
 		log(chalk.blackBright(`Data: ${dtHrCriado}`));
-		log(chalk.blackBright(`Local: ${unidade.nome}`));
+		log(chalk.blackBright(`Local: ${unidade?.nome}`));
 
 		if (unidadeDestino) {
 			log(chalk.blackBright(`Indo para: ${unidadeDestino?.nome}`));
 		}
 
 		log();
 	});
+	return null;
```

Comment:
qual o motivo de adicionar esse return?

Reply:
Eu também achei que era desnecessário em um primeiro momento, mas o ESLint mostra o seguinte erro:
`Expected to return a value at the end of async function 'run'.eslintconsistent-return`

E sugere a seguinte prática para resolver esse problema:
https://eslint.org/docs/rules/consistent-return